platform/src/server/http/http_server.ts

```diff
@@ -2,7 +2,7 @@ import express from 'express';
 import http from 'http';
 import https from 'https';
 import { Server } from 'net';
-import { promisify } from 'bluebird';
+import { promisify } from 'util';
```

Comment:
__nit:__ I can't recall why I was using `promisify(....).call(context, ...)` pattern in the first place, but it seems it's not longer needed with node's native `promisify` and we can just use `promisify(...)(...)`. Would you mind updating two places in this file?

Reply:
Looks like it fails for the `server.listen` case with:

```
[ts] Expected 0 arguments, but got 2.
```

Reply:
Yeah, it probably wants something like this `promisify<number, string, Function>`, but I just checked that `listen` still needs a proper `this` and `promisify` doesn't allow to set this, so just ignore my comment, sorry for the noise.